be/src/storage/tablet_updates.cpp

```diff
@@ -164,6 +164,7 @@ Status TabletUpdates::_load_from_pb(const TabletUpdatesPB& tablet_updates_pb) {
     // Load all rowsets of this tablet into memory.
     // NOTE: This may change in a near future, e.g, manage rowsets in a separate module and load
     // them on demand.
+    _rowsets.clear();
```

Comment:
It's better to add commit message to describe the problem and solution

Reply:
updated